microsoft-edge/webview2/how-to/print.md

```diff
@@ -349,7 +348,7 @@ CoreWebView2PrintSettings GetSelectedPrinterPrintSettings(string printerName)
 
   return printSettings;
 
-  // or
+  // or, you can do the following:
```

Comment:
@monica-ch - is this the intent of the "// or" line?

Reply:
For "or"

Reply:
reverted 3 lines by Find: // or